Tasks/FileTransform/filetransform.ts

```diff
@@ -0,0 +1,26 @@
+import tl = require('vsts-task-lib/task');
```

Comment:
The name of the task must be appended with the Version number. FileTransformV0

Reply:
Also, add the task name in the make-options file.

Reply:
Got it